SU2_CFD/include/solvers/CIncNSSolver.hpp

```diff
@@ -157,6 +157,22 @@ class CIncNSSolver final : public CIncEulerSolver {
                           CConfig *config,
                           unsigned short val_marker) override;
 
+  /*!
+   * \brief Impose a heat flux by prescribing a heat transfer coefficient and a wall temperature.
+   * \param[in] geometry - Geometrical definition of the problem.
+   * \param[in] solver_container - Container vector with all the solutions.
+   * \param[in] conv_numerics - Description of the numerical method.
+   * \param[in] visc_numerics - Description of the numerical method.
+   * \param[in] config - Definition of the particular problem.
+   * \param[in] val_marker - Surface marker where the boundary condition is applied.
+   */
+  void BC_HeatTransfer_Wall(CGeometry *geometry,
+                            CSolver **solver_container,
+                            CNumerics *conv_numerics,
+                            CNumerics *visc_numerics,
+                            CConfig *config,
```

Comment:
Can we constify some of these? or even better remove what we might (I did not check) not need?

Reply:
👍 solver_container and *_numerics are gone now and the remaining inputs are const